sdk/textanalytics/azure-ai-textanalytics/samples/sample_multiple_analysis.py

```diff
@@ -0,0 +1,159 @@
+from azure.core.credentials import AzureKeyCredential
```

Comment:
Hey @nawta thanks for your PR. There is a sample that shows multiple analysis of actions here: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/textanalytics/azure-ai-textanalytics/samples/sample_analyze_actions.py

It intentionally doesn't capture all *Action input types, though, just a few of the ones that use prebuilt models so the output is not overwhelming. It's also hard to come up with input text that captures interesting results for all actions, e.g. input text that works well with sentiment, might not capture anything for healthcare. To set this sample apart from that one, what if we make this sample focus on the custom actions? The custom actions require a bit more upfront effort to train a model and I'm not sure we want to include them together with the prebuilt models which only require your endpoint/key to run. Let me know what you think! :)

Reply:
Oh my, I did not notice there was already one. Thank you for letting me know....

> what if we make this sample focus on the custom actions?
That sounds great! Let me modify the code :)
So we gonna narrow down the actions to RecognizeCustomEntitiesAction, SingleLabelClassifyAction, and MultiLabelClassifyAction?

Reply:
Btw, just a subtle proposal. How about changing the file name of https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/textanalytics/azure-ai-textanalytics/samples/sample_analyze_actions.py to sample_multiple_analysis.py or something?
For, some goofy like me may miss the Multiple Analysis samples because I looked for the word "multiple" in the directory xD

Reply:
Yes, I think that sounds good.

Reply:
> sorry I commented on this before I saw your second comment. Let's hold off on the name change for now since I need to check how it would affect articles linking to the sample. I don't want to create 404s by accident. That is good feedback, however, so I'll double-check and get back to you.

Yes of course! It's just an opinion. No pressure.